docs/public-identity-infra/standard.md

```diff
@@ -0,0 +1,117 @@
+# Public Modding Identity Infrastructure
+
+## 1. Introduction
+
+This document defines the infrastructure used to distribute code certificates to Minecraft mod developers and the policies surrounding this system.
+
+### 1.1. Terminology
+
+ The keywords "MUST", "MUST NOT", "SHOULD", and "MAY" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## 2. Certificate Authorities
+
+Certificate Authorities are organisations responsible for maintaining, delivering and invalidating code certificates.
+
+### 2.1. Root Authorities
+
+Root Authorities are Certificate Authorities that can be fully trusted by verifiers. They MAY deliver certificates directly and MAY sign other Certificate Authorities after verifying the new Certificate Authority is conforming to this document.
+
+### 2.2. Authority Components
+
+Certificate Authorities MUST provide the following component:
+- Online Certificate Status Protocol server ([RFC 6960](https://www.rfc-editor.org/rfc/rfc6960))
+
+Certificate Authorities SHOULD and Root Authorities MUST provide the following components:
+- Time-Stamp Protocol server ([RFC 3161](https://www.rfc-editor.org/rfc/rfc3161))
+- Certificate Transparency log ([RFC 6962](https://www.rfc-editor.org/rfc/rfc6962))
+
+### 2.3. Certificate Safety
+
+Certificate Authorities MUST NOT sign certificates directly using their primary certificate. Rather, one or multiple intermediary certificate valid for equal or less than one year MUST be used. The primary certificate MUST be stored offline and SHOULD be stored in a Hardware Security Module.
+
+## 3. Certificate Structure
+
+Certificate MUST use the X.509 format as defined by [RFC 5280](https://www.rfc-editor.org/rfc/rfc5280). 
+
+### 3.1. Subject Fields
+
+#### 3.1.1. Common Name (CN)
+
+The subject Common Name MUST be the subject real world name or a uniquely identifiable username. Under certain circumstances, the Common Name MAY be set to `Automation` if the Organization field is set.
+
+#### 3.1.2. Organization (O)
+
+The subject Organization CAN be the subject organisation under which that certificate is applicable. It MAY be set to `Developer` if not applicable.
+
+#### 3.1.3. Distinguished Name (DN)
+
+The subject Distinguished Name MUST be constructed according to [RFC 5280](https://www.rfc-editor.org/rfc/rfc5280), using the `CN` and `O` fields.
```

Comment:
It might be worth mentioning what sections you're referring to, given the table of contents doesn't seem to point at a basic idea of what a Distinguished Name is, or what it's for.